### PR TITLE
Intro-Python-II - MVP

### PR DESCRIPTION
**Pull request by vebradev**

### CODE REVIEW

File: src/adv.py [hunk 1]
```diff
@@ -37,15 +38,75 @@
...
 # If the user enters "q", quit the game.
+
+while status:
+    print(f"\n{player}")
+
+    direction = input(messages["dir_input"]).upper()
+
+    if len(direction) >= 1:
+        if direction[0] == "N":
+            if hasattr(player.location, "n_to"):
```
Comment (openwell) [inline src/adv.py]: good use of the method hasattr. you didn't go down the laid of using getattr by have self.n_to and others in the player class

**Review by openwell — COMMENTED** [review]
Great job so far. 
You demonstrated your understanding of python very well.
Git workflow was good.

**Review by openwell — COMMENTED** [review]
A job well done... I see you haven't slept.
I guess you are yet to be done.
....list of room items
.....list of player items yet to be implemented

**Comment by vebradev** [comment]
> A job well done... I see you haven't slept.
> I guess you are yet to be done.
> ....list of room items
> .....list of player items yet to be implemented

Hey @openwell! I guess I'm not the only one who's still up 😅. Refactored `adv.py`, fleshed out classes, rendering room/player items on screen and functionality to take/drop them in and from rooms is here. Also, game is winnable. Try without looking at source!

I know that code looks hairy and messy and all, but I'm kind of happy to get this done knowing it's the first week of Python. Will do better next time.

**Review by openwell — APPROVED** [review]
Kudos on the refactoring and achieving the MVP

**Pull request merged**